crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -327,120 +209,37 @@ impl SignedCheckpointSummary {
 // or other authenticated data structures to support light
 // clients and more efficient sync protocols.
 
+pub type CheckpointSummaryEnvelope<S> = Envelope<CheckpointSummary, S>;
 pub type CertifiedCheckpointSummary = CheckpointSummaryEnvelope<AuthorityStrongQuorumSignInfo>;
+pub type SignedCheckpointSummary = CheckpointSummaryEnvelope<AuthoritySignInfo>;
 
-impl CertifiedCheckpointSummary {
-    /// Aggregate many checkpoint signatures to form a checkpoint certificate.
-    pub fn aggregate(
```

Comment:
don't entirely follow all the changes to the signature logic, maybe a second pair of eyes on that would be worthwhile (or could walk me through it on huddle?)

Reply:
sorry this is from a stacked PR, let me rebase